components/script_bindings/Cargo.toml

```diff
@@ -16,6 +16,7 @@ name = "script_bindings"
 path = "lib.rs"
 
 [dependencies]
+ascii = { workspace = true }
```

Comment:
Is this used in script_bindings? I can't see it anywhere.

Reply:
The two conversion functions at the top use asciistring. I didn't want to implement them myself for fear of doing something wrong.